source/crossdomain.xml

```diff
@@ -4,7 +4,8 @@
     <!-- Read this: www.adobe.com/devnet/articles/crossdomain_policy_file_spec.html -->
 
     <!-- Most restrictive policy: -->
```

Comment:
Can you please update the comment here to reflect what this does?  Just change it so it says something like "only allow https cross domain requests from api.github.com" or something

Reply:
Done, Opted to be extra verbose as that's how I like my comments. :D

Reply:
Is there a reason for us to update the crossdomain.xml file? Only Flash player would use it. Github's API already sends all the necessary CORS headers.

Reply:
I kinda wondered the same, In my internal tests everything worked perfect without the changes to the crossdomain stuff, which is why I didn't initially put it in.
I don't know much about nor understand crossdomain files fully atm.

On the flip side the change won't have a negative impact on security since we're only allowing stuff from api.github etc. I think.